web/blog/2022-06-16-vast-v2.1/README.md

```diff
@@ -0,0 +1,41 @@
+---
+draft: true
```

Comment:
Note for the reviewer: This option makes it so the development version of the docs shows this blog post, but it's hidden when building a production version of the docs.